package.json

```diff
@@ -78,7 +78,8 @@
     "react-scripts": "5.0.1",
     "react-useportal": "1.0.19",
     "redux": "5.0.1",
-    "vanilla-framework": "4.5.1"
+    "vanilla-framework": "4.5.1",
+    "yup": "1.3.3"
```

Comment:
This is the validation library that gets used in our other projects that use Formik (I was surprised to see it wasn't being used already).